jest.config.js

```diff
@@ -32,12 +34,5 @@ module.exports = {
       statements: 100,
     },
   },
-  coveragePathIgnorePatterns: ["/node_modules/", "<rootDir>/dist"],
```

Comment:
I'm not sure why specifying a specific coverage reporter changes the paths that should be ignored measuring coverage. Does this config has an impact in the current setting ? And if so can you elaborate on why it isn't useful anymore ?

Reply:
The paths `node_modules`is ignored by default, the `dist` folder would not be picked up by unit tests anyway, so that was unnecessary anyway.

As you can see, the test coverage is accurately reported: https://github.com/inrupt/solid-client-js/runs/4521157986?check_suite_focus=true#step:7:70